src/morpho-blue/MorphoBlueSnippets.sol

```diff
@@ -59,12 +56,11 @@ contract MorphoBlueSnippets {
         (uint256 totalSupplyAssets,, uint256 totalBorrowAssets,) = morpho.expectedMarketBalances(marketParams);
 
         // Get the borrow rate
-        uint256 borrowRate = borrowAPY(marketParams, market);
-
-        // Get the supply rate
-        uint256 utilization = totalBorrowAssets == 0 ? 0 : totalBorrowAssets.wDivUp(totalSupplyAssets);
-
-        supplyRate = borrowRate.wMulDown(1 ether - market.fee).wMulDown(utilization);
+        if (marketParams.irm != address(0)) {
```

Comment:
the previous returned value was correct since `borrowRate` when `(marketParams.irm != address(0))`, no?

Reply:
nope as the borrowAPY returns an already compounded formula while for the supply the approximation is done after:

<img width="998" alt="image" src="https://github.com/morpho-org/morpho-blue-snippets/assets/75627352/6c898cc2-e6ea-42cf-958f-c1a85a199b23">

cf: https://docs.morpho.org/contracts/morpho-blue/reference/irm/

Reply:
nvm I misread `borrowRate` and `borrowAPY`